dist/index.html

```diff
@@ -54,19 +54,19 @@ <h3>Create New Booking</h3>
             <h4>Filter by Room Type:</h4>
             <section class="room-filters" id="room-filters">
               <div class="single-filter">
-                <input class="checkbox" type="checkbox" id="single-room" value="single-room">
+                <input class="checkbox-js" type="checkbox" id="single-room" value="single-room">
```

Comment:
Make sure to change all your id's to camelCase!!!

Reply:
just added the to my to-do list!